python/sglang/srt/models/deepseek_v2.py

```diff
@@ -187,8 +189,8 @@ def __init__(
             else (EPMoE if global_server_args_dict["enable_ep_moe"] else FusedMoE)
         )
         self.experts = MoEImpl(
-            num_experts=config.n_routed_experts,
-            top_k=config.num_experts_per_tok,
+            num_experts=config.n_routed_experts + self.n_share_fusion_experts,
+            top_k=config.num_experts_per_tok + min(self.n_share_fusion_experts, 1),
```

Comment:
Why `n_share_fusion_experts` is clipped? How to set the gating weight for the extra experts?

Reply:
Since the shared experts copied here are all the same expert, only one additional topk is needed, that is, from top8 to top9.